types/evidence.go

```diff
@@ -19,12 +19,12 @@ import (
 
 const (
 	// MaxEvidenceBytes is a maximum size of any evidence (including amino overhead).
-	MaxEvidenceBytes        int64 = 436
-	ValidatorsHashField           = "ValidatorsHash"
-	NextValidatorsHashField       = "NextValidatorsHash"
-	ConsensusHashField            = "ConsensusHash"
-	AppHashField                  = "AppHash"
-	LastResultsHashField          = "LastResultsHash"
+	MaxEvidenceBytes        int64  = 436
+	ValidatorsHashField     string = "ValidatorsHash"
```

Comment:
?? why add the type

Reply:
linter was complaining that maxEvidenceBytes is the only one with type so added them below as well.